src/pages/components/StopSelector.tsx

```diff
@@ -20,7 +19,7 @@ const StopSelector = ({ stops, stopKey, setStopKey }: StopSelectorProps) => {
       value={value}
       onChange={(e, value) => setStopKey(value ? value.key : '0')}
       id="stop-select"
-      sx={{ width: INPUT_SIZE * 2 }}
+      sx={{ width: '100%' }}
```

Comment:
The "StopSelector" component uses input field the same as "RouteSelector" and "OperatorSelector" so you don't need here to set the width